### PR TITLE
EN: Update co_031417.csv

### PR DESCRIPTION
**Pull request by SanseiMuramasa**
Tutorial translation: Proposed JP > EN translation for 「パートナー好感度」 into "Partner Evaluation"

### CODE REVIEW

**Comment by alama** [comment]
Can you please move the changed file to `Tutorial/Window` folder

**Comment by SanseiMuramasa** [comment]
I will move it to Tutorial/Window

File: Tutorial/Window/co_031417.csv [hunk 1]
```diff
@@ -0,0 +1,3 @@
+text00#0,"""Partner Evaluation"""
+text01#0,"""Partner Evaluation<br><br><br>If you clear a quest<br>with a partner character,<br>Or if you complete<br>one of their client orders,<br> That \""Partner Evaluation\"" of you will go up.<br><br>A Partner's \""Partner Evaluation\"" can be found in the campship<br>from the \""Partner Terminal\""<br> from the \""Partner List\""<br>"""
```
Comment (alama) [inline Tutorial/Window/co_031417.csv]: "Campship", not "campship"

File: Tutorial/Window/co_031417.csv [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+text02#0,"""As your partner's \""Partner Evaluation\"" goes up<br>the \""Present Gague\"" will also go up.<br><br>When that \""Present Gague\"" maxes out<br>That character's will give you<br>their present.<br><br>A \""present icon\"" will show above the partner's head.<br><br>A \""present icon\"" is shown beside \""Talk\"".<br>Press \""Talk\""<br>to get the present."""
```
Comment (alama) [inline Tutorial/Window/co_031417.csv]: "Gauge", not "Gague"

**Review by alama — CHANGES_REQUESTED** [review]
the English patch goes for EN-US spelling

**Review by alama — APPROVED** [review]

**Comment by SynthSy** [comment]
Needs to be proofread.

**Comment by SanseiMuramasa** [comment]
Changed "That character's will give you their present" to "That character will give you their present".

**Comment by SanseiMuramasa** [comment]
Re-doing the translation for 好感度, as I realized it's not in JP brackets. 「」

I overused the term "Partner Evaluation" in brackets for  好感度. This will be replaced with "character's evaluation" "their evaluation" or "evaluation of you" etc. depending on the context of the situation, plus with no EN brackets.

This will make more sense in plain English as you will see later.

**Comment by SanseiMuramasa** [comment]
Re-translating 好感度 as "Friendship level", as it seems to be a more appropriate translation.

File: Tutorial/Window/co_031417.csv [hunk 3]
```diff
@@ -0,0 +1,3 @@
+text00#0,"""Partner's Friendship Level"""
+text01#0,"""Partner's Friendship Level <br><br><br>If you clear a quest<br>with a partner character,<br>Or if you complete<br>one of their client orders,<br> That character's Friendship Level will go up.<br><br>Your partner's Freiendship Level can be found in the Campship<br>from the \""Partner Terminal\""<br> from the \""Partner List\"".<br>"""
```
Comment (alama) [inline Tutorial/Window/co_031417.csv]: "Freiendship", Should it be "Friendship" like a set of ponies?

**Comment by SanseiMuramasa** [comment]
The spelling mistake for "Friendship" has been fixed.

File: Tutorial/Window/co_031417.csv [hunk 4]
```diff
@@ -0,0 +1,3 @@
+text00#0,"""Partner's Friendship Level"""
+text01#0,"""Partner's Friendship Level <br><br><br>If you clear a quest<br>with a partner character,<br>Or if you complete<br>one of their client orders,<br> That character's Friendship Level will go up.<br><br>Your partner's Friendship Level can be found in the Campship<br>from the \""Partner Terminal\""<br> from the \""Partner List\"".<br>"""
```
Comment (SynthSy) [inline Tutorial/Window/co_031417.csv]: Or shouldn't be capitalized.

File: Tutorial/Window/co_031417.csv [hunk 5]
```diff
@@ -0,0 +1,3 @@
...
+text02#0,"""As your partner's Friendship Level goes up<br>the \""Present Gauge\"" will also go up.<br><br>When that \""Present Gauge\"" maxes out<br>That character will give you<br>their present.<br><br>A \""present icon\"" will show above the partner's head.<br><br>A \""present icon\"" is shown beside \""Talk\"".<br>Press \""Talk\""<br>to get the present."""
```
Comment (SynthSy) [inline Tutorial/Window/co_031417.csv]: ` maxes out<br>That` should be `maxes out, <br>that`

Comment (SanseiMuramasa) [inline Tutorial/Window/co_031417.csv]: One more thing.

I translated プレゼントマーク as "Present Icon" because "Present Mark" doesn't seem to make sense in English.

Is this okay?
